core/shared/src/main/scala/org/http4s/headers/Forwarded.scala

```diff
@@ -203,7 +204,7 @@ object Forwarded extends ForwardedRenderers {
       */
     def fromString(s: String): ParseResult[Host] =
       parser.parseAll(s).left.map { e =>
-        ParseFailure(s"invalid host '$s'", e.toString)
+        ParseFailure(s"invalid host '$s'", e.show)
```

Comment:
Thanks, so then my question is why aren't we using `fromParser` here? Sorry if it's dumb one 😅

Reply:
🤷🏻  probably this could be refactored